src/ccpp_types.F90

```diff
@@ -20,9 +20,7 @@
 module ccpp_types
 
 !! \section arg_table_ccpp_types
-!! | local_name                        | standard_name             | long_name                                             | units   | rank | type      |   kind   | intent | optional |
-!! |-----------------------------------|-------------------------- |-------------------------------------------------------|---------|------|-----------|----------|--------|----------|
-!! | ccpp_t                            | ccpp_t                    | definition of type ccpp_t                             | DDT     |    0 | ccpp_t    |          | none   | F        |
+!! \htmlinclude ccpp_types.html
```

Comment:
This change requires updates to the Technical Documentation (chapter 6) to reflect that ccpp_types.F90 no longer has a table. Let me know if you have time to do these changes or I can do them.

Reply:
Yes, please, make those changes if you can! Thanks, Dom